canonical-web-and-design/charmhub.io
# Consume libraries API

## Done

- Remove mocked data
- Consume libraries API

Fixes #518

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check any charm `/libraries`
